tests/test_merge.py

```diff
@@ -279,4 +279,44 @@ def test_merge_branch_skip_all(wst, capsys):
         assert not os.path.isfile(os.path.join(temp_git_dir_path, "commit3.xml"))
 
     out, _ = capsys.readouterr()
-    assert out.split('\n')[0] == 'Merging 3.0.x into master'
\ No newline at end of file
+    assert out.split('\n')[0] == 'Merging 3.0.x into master'
+
+
+def test_merge_branch_with_user(wst, capsys):
+    """
+        Test to check if merge_branch works with user option
```

Comment:
You shouldn't need this extra indentation.  You should try out yapf for auto format correction.

Reply:
Read this comment after the PR was merged. Created a follow up PR to address the extra indentations: https://github.com/confluentinc/workspace-tools/pull/9